apps/web-giddh/src/app/invoice/preview/models/bulk-export-modal/bulk-export.component.ts

```diff
@@ -0,0 +1,140 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { BulkVoucherExportService } from 'apps/web-giddh/src/app/services/bulkvoucherexport.service';
+import { GeneralService } from 'apps/web-giddh/src/app/services/general.service';
+import { ToasterService } from 'apps/web-giddh/src/app/services/toaster.service';
+import { EMAIL_VALIDATION_REGEX } from 'apps/web-giddh/src/app/app.constant';
+
+@Component({
+    selector: 'invoice-bulk-export',
+    templateUrl: './bulk-export.component.html',
+    styleUrls: [`./bulk-export.component.scss`]
+})
+
+export class BulkExportModal implements OnInit {
+    /** Type of voucher */
+    @Input() public type: string = "sales";
+    /** Selected Vouchers */
+    @Input() public voucherUniqueNames: any[] = [];
+    /** Advance Search Parameters */
+    @Input() public advanceSearch: any;
+    /** From/To Date On Page */
+    @Input() public dateRange: any;
+    /** Total Items For Export */
+    @Input() public totalItems: any = 0;
+    /** Emit the close modal event */
+    @Output() public closeModelEvent: EventEmitter<boolean> = new EventEmitter(true);
+
+    /** Download Voucher Copy Options */
+    public downloadCopyOptions: any[] = [
+        { label: 'Original', value: 'ORIGINAL' },
+        { label: 'Customer', value: 'CUSTOMER' },
+        { label: 'Transport', value: 'TRANSPORT' }
+    ];
+
+    /** Selected download Voucher Copy Options */
+    public copyTypes: any = [];
+    /** Email Receivers */
+    public recipients: any = "";
+    /** Will handle if api call is in process */
+    public isLoading: boolean = false;
+
+    constructor(private bulkVoucherExportService: BulkVoucherExportService, private generalService: GeneralService, private toaster: ToasterService) {
```

Comment:
Consider breaking the injected dependencies in multiple lines for better readability